### PR TITLE
Add new props

### PR DESCRIPTION
**Pull request by romain130492**
Hello !
Thanks for this Vue-Map
I added new props in your repository, : legendFontColorHeader, legendFontColorContent,legendBoxShadow ,legendHeaderBackgroundColor,legendContentBackgroundColor,chromaScaleOn 

I also documented the New Props in the ReadMe
I hope it could be usefull !
I stay available for any changes if needed,

### CODE REVIEW

**Comment by romain130492** [comment]
i just need to re-format the readMe, some issue in here apparently

**Comment by romain130492** [comment]
> i just need to re-format the readMe, some issue in here apparently

The readMe has been updated normally

File: src/App.vue [hunk 1]
```diff
@@ -16,121 +17,165 @@
...
+        this.$props.lowColor,
+        this.$props.highColor,
+        this.$props.defaultCountryFillColor,
+        this.$props.legendHeaderBackgroundColor,
+        this.$props.legendContentBackgroundColor,
+        this.$props.legendFontColorHeader,
+        this.$props.legendFontColorContent,
+        this.$props.legendBorderColor,
+        this.$props.legendBorderRadius,
+        this.$props.legendBoxShadow
```
Comment (maguayo) [inline src/App.vue]: I think this is the problem. The **chroma.scale** has to contain two values, the lowColor and highColor (https://github.com/gka/chroma.js/).

**Comment by romain130492** [comment]
All right, I fixed it by removing the new parameters from the function ChromaScale Now it should work,!

**Pull request merged**